### ISSUE TITLE
Show ILvl with mouse over

### ISSUE THREAD

**Issue by e8ghtball** [kolton/d2bot-with-kolbot#1398]
How do I make Kolbot show the ILVL when I mouse over a item

**Comment by IamLostPlzHelpMe** [comment]
i dont think you can, i think the only way is to look at  the item in the console, then you can see "matriarchal javelin (88)" (item lvl88)

**Comment by DarkHorseDre** [comment]
This is done by default when you start with `D2BotMap.dbj` (I think) - but if you want to do it ingame you may need to type into console as a quick solution.

Try this in console - it should give the item detail, where level will be one parameter:
JSON.stringify(getUnit(101));

I cant look into it now but there is a way to specify the level (print(getUnit(101).level or .ilvl) ?)

**Comment by 5noop** [comment]
Set UserAddon to true and start the bot with D2BotBlank.

**Comment by DarkHorseDre** [comment]
oops - I need to stop posting when half asleep :)

damn you on-point @5noop  :)

**Comment by DW71** [comment]
this feature is really needed for cubing. if it is too difficult to show ilvl with mouse hover, is it possible to bind this feature  on a special button.?

**Comment by DarkHorseDre** [comment]
So you can do it as it stands whilst playing manually as @5noop said.

Do you want to only cube items of a certain level? do you want to do this with the bot or only when playing manually? (the bot can do it with modification)

tell us your use case and we'll help you out

**Comment by DW71** [comment]
@DarkHorseDre Thanks for reply, 1) Yes I do manual cubing and I heard that only high level item can give you the possibility of nice attributes. 2) some players really believe and collect a certain ilvl items for lucky or superstition ;-) whatever, the bot is really nice!

**Comment by DarkHorseDre** [comment]
haha ok, so you should do what @5noop said - the item level (and other data) will show when you set `Scripts.UserAddon = true` (I always forget that ffs).

as for item level - it is important for rolling in nearly all cases, see http://classic.battle.net/diablo2exp/items/magic/

The area (or boss) level usually dictates the item level so pay attention to where you kill mobs.
Theres a better list I used but cant find it - check this: https://github.com/kolton/d2bot-with-kolbot/blob/master/d2bs/kolbot/sdk/areas.txt

that said, I would consider setting the bot to cube - it is faster as it will find all materials and items of correct levels (although for magics, it fixes the level range, but you can override it with a small change)

**Comment by DW71** [comment]
@DarkHorseDre You rock ~ ! 👍

**Comment by iihieiii** [comment]
> haha ok, so you should do what @5noop said - the item level (and other data) will show when you set `Scripts.UserAddon = true` (I always forget that ffs).
> 
> as for item level - it is important for rolling in nearly all cases, see http://classic.battle.net/diablo2exp/items/magic/
> 
> The area (or boss) level usually dictates the item level so pay attention to where you kill mobs.
> Theres a better list I used but cant find it - check this: https://github.com/kolton/d2bot-with-kolbot/blob/master/d2bs/kolbot/sdk/areas.txt
> 
> that said, I would consider setting the bot to cube - it is faster as it will find all materials and items of correct levels (although for magics, it fixes the level range, but you can override it with a small change)

nice! wich script should i use to manually mule items with another cd-key ? Also d2botblank with useraddon true ?? thx in adv ;)
